[StructuralMechanics] Explicit Solid test failing

~~~
======================================================================
FAIL: test_execution (structural_mechanics_test_factory.ExplicitSolidBeam)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_prod/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 55, in test_execution
    self.test.RunSolutionLoop()
  File "/home/philippb/software/Kratos_prod/kratos/python_scripts/analysis_stage.py", line 66, in RunSolutionLoop
    self.FinalizeSolutionStep()
  File "/home/philippb/software/Kratos_prod/kratos/python_scripts/analysis_stage.py", line 141, in FinalizeSolutionStep
    process.ExecuteFinalizeSolutionStep()
  File "/home/philippb/software/Kratos_prod/kratos/python_scripts/check_scalar_to_nodes_process.py", line 124, in ExecuteFinalizeSolutionStep
    self.assertAlmostEqual(self.value, value, self.tolerance_rank)
AssertionError: -1.664348560603663e-06 != -1.1640973078099457e-06 within 6 places
~~~

guess this started to fail during your recent refactorings @loumalouomega 

Note: this test is in the validation queue

--- a/applications/StructuralMechanicsApplication/tests/explicit_tests/explicit_solid_beam_test_parameters.json
+++ b/applications/StructuralMechanicsApplication/tests/explicit_tests/explicit_solid_beam_test_parameters.json
@@ -69,7 +69,7 @@
                     "model_part_name" : "Check_ModelPart",
                     "variable_name"   : "DISPLACEMENT_Y",
                     "interval"        : [0.0, "End"],
-                    "value"           : "-3.28e-5*(1.0 - cos((3.14159/0.0049)*t))",
+                    "value"           : "-3.28e-5*(1.0 - cos((3.14159/0.0049)*(t - 1.0e-6)))",
                     "tolerance_rank"  : 6
             }
         }
@@ -105,6 +105,26 @@
                     },
                     "point_data_configuration"  : []
                 }
+            }
+        }],
+        "vtk_output" : [{
+            "python_module" : "vtk_output_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "VtkOutputProcess",
+            "help"          : "This process writes postprocessing files for Paraview",
+            "Parameters"    : {
+                "model_part_name"                    : "Structure.computing_domain",
+                "output_control_type"                : "step",
+                "output_frequency"                   : 1,
+                "file_format"                        : "ascii",
+                "output_precision"                   : 7,
+                "output_sub_model_parts"             : true,
+                "folder_name"                        : "vtk_output",
+                "save_output_files_in_folder"        : true,
+                "nodal_solution_step_data_variables" : ["DISPLACEMENT","REACTION","VELOCITY","ACCELERATION"],
+                "nodal_data_value_variables"         : [],
+                "element_data_value_variables"       : [],
+                "condition_data_value_variables"     : []
             }
         }]
     }

